fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1990,10 +1974,10 @@ protected boolean replayJournalInner(JournalCursor cursor, boolean flowControl)
                 // apply
                 EditLog.loadJournal(this, entity);
             } catch (Throwable e) {
-                if (canSkipBadReplayedJournal()) {
+                if (canSkipBadReplayedJournal(e)) {
                     LOG.error("!!! DANGER: SKIP JOURNAL {}: {} !!!",
                             replayedJournalId.incrementAndGet(),
-                            entity == null ? null : entity.getData(),
+                            entity == null ? null : GsonUtils.GSON.toJson(entity.getData()),
```

Comment:
need to cap the size of the result of toJson

Reply:
Better to print the entire message to make debugging the problem easier.